src/easylogging++.h

```diff
@@ -4008,10 +4008,23 @@ class VRegistry : base::NoCopy, public base::threading::ThreadSafe {
     return !base::utils::hasFlag(LoggingFlag::DisableVModules, *m_pFlags);
   }
 
+  /// @brief Set replacement name for thread id
+  void setThreadName(const char* name) {
```

Comment:
this is in wrong spot, vmodules is for verbose levels and all related to verbose logs. I like it but the best spot is under Storage class as that holds everything